libs/data-mapper-v2/src/components/commandBar/EditorCommandBar.tsx

```diff
@@ -56,10 +58,11 @@ export const EditorCommandBar = (_props: EditorCommandBarProps) => {
     (state: RootState) => state.modal.warningModalType === WarningModalState.DiscardWarning && state.modal.isOkClicked
   );
 
-  const dataMapDefinition = useMemo<string>(() => {
+  const dataMapDefinition = useMemo<MetaMapDefinition>(() => {
```

Comment:
Not really using this yet, but need to get consensus from the team on how we handle this scenario long-term before removing! Don't want to duplicate any work